pom.xml

```diff
@@ -1783,6 +1806,21 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
                 <version>${maven.enforcer.plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>[3.3.1,3.3.9]</version>
```

Comment:
This requirement must be documented on https://github.com/mulesoft/mule/blob/mule-4.x/CONTRIBUTING.md